### ISSUE TITLE
Incorrect value being passed on method

### ISSUE THREAD

**Issue by gaurav2493** [slackapi/java-slack-sdk#378]
### Issue Type

- [X] Bug
- [ ] Enhancement / Feature request
- [ ] Question
- [ ] Documentation

### Description

While Implementing the Interface `InstallationService`,  I observed that the method 
`Bot findBot(String enterpriseId, String teamId) `
 was getting incorrect value for `teamId`.  `teamId` actually had the value for `enterprise_id`
when an event API for `message.im` came.

I figured out that in `EventRequest.java`, line 35, the following code is causing the problem
```
if (enterpriseId != null) {
   this.getContext().setTeamId(enterpriseId.getAsString());
 }
```
---

The issue is reproducible in:
* Module Version: 1.0.0-M3-SNAPSHOT
* JDK Version: 1.8.0_231

The steps to reproduce are:
1. Subscribe to `message.im`
2. Create a distributed app
3. Send a direct message to the Chat bot

The expected result is:
context.teamId and contest.enterpriseId to contain teamId and enterpriseId respectively

The actual result is:
both context.teamId and contest.enterpriseId have enterprise_id as their values.

---

### Requirements (place an `x` in each of the `[ ]`)

**TODO**:
* [x] I've read and understood the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

**Comment by seratch** [comment]
@gaurav2493 Hello, thank you for pointing out this issue! Oops, this is a bit silly mistake... I'll check all the other classes again. 🙇

**Comment by seratch** [comment]
I've reviewed all the setter calls and didn't see similar bugs so far.

**Comment by seratch** [comment]
Closed by #379, thanks!
